src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/BlockDataExtractor.java

```diff
@@ -48,19 +47,15 @@ public void generateJsonFile() throws IOException {
             writeOutTransactions(start, seqWriter);
         }
         seqWriter.close();
-
     }
 
     public void writeOutTransactions(int blockNumber, SequenceWriter seqWriter) throws IOException {
         BigInteger blockBigInteger = BigInteger.valueOf(blockNumber);
-
-        EthBlock.Block ethBlock = client.getEthBlock(blockBigInteger).getBlock();
-
-        List transList = ethBlock.getTransactions();
-        ListIterator it = transList.listIterator();
+        BlockWithData block = client.getBlockWithData(blockBigInteger);
+        List transactions = block.getTransactions();
```

Comment:
Don't remove all blank lines that might have been used for prior logical grouping (e.g. it seemed sensible to have the retrieval of the iterator as part of the logical while block).

The sensible refactoring here would be to replace the iterator-while approach, this is not idiomatic for modern Java anymore.

Reply:
It is not my priority, of course, doing so in this branch. But I need to have a coding style to have a professional code. Without it, I am using a consistent one defined by myself.

Reply:
Just follow something along [Google Java Codestyle](https://google.github.io/styleguide/javaguide.html). 

Also just use auto format with IntelliJ default style (depends on shortcut config, but normally `Ctrl+Shift+Alt+L`,  should be fine for most of the cases https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html).

It seems not sensible to remove **all** blank lines in a method body, just want to make sure this is not an implicit coding style you are targeting. There are (and can't be) hard coding styles about when to use vertical space for organizing code logically. 

I think a good logical grouping could be the following:

```java

    public void writeOutTransactions(int blockNumber, SequenceWriter seqWriter) throws IOException {
        BigInteger blockBigInteger = BigInteger.valueOf(blockNumber);
        EthBlock.Block ethBlock = client.getEthBlock(blockBigInteger).getBlock();

        List transList = ethBlock.getTransactions();

        ListIterator it = transList.listIterator();
        while (it.hasNext()) {
            seqWriter.write(new ReducedTransObject((Transaction) it.next()));
        }
    }
```

Coding style in a team is an emergent property, therefore no other way but discuss here as part of PRs.